plugins/smart_proxy_dynflow/postinst

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+# postinst script for smart_proxy_dynflow
+#
+# see: dh_installdeb(1)
+
+set -e
+
+USERNAME="foreman-proxy"
+GROUPNAME="foreman-proxy"
+DYNFLOW_DB_DIR="/var/lib/foreman-proxy/dynflow"
+
+case "$1" in
+  configure)
+    mkdir -p $DYNFLOW_DB_DIR
+    chown $USERNAME:$GROUPNAME $DYNFLOW_DB_DIR
```

Comment:
Could you also set a restrictive mode for the DB here?  It'll prevent any user reading the DB.